python/tach/constants/__init__.py

```diff
@@ -7,7 +7,13 @@
 ROOT_MODULE_SENTINEL_TAG = "<root>"
 TACH_YML_SCHEMA_URL = "https://raw.githubusercontent.com/gauge-sh/tach/v0.10.0/public/tach-yml-schema.json"
 
-DEFAULT_EXCLUDE_PATHS = ["tests", "docs", "venv", ".*__pycache__", ".*egg-info"]
+DEFAULT_EXCLUDE_PATHS = [
```

Comment:
The default exclude paths should match the default of `use_regex_matching` in the project config. Given that, I don't think we should update this yet.

Reply:
True, missed this from older commits